manifests/c/craftablescience/VPKEdit/4.0.0/craftablescience.VPKEdit.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: craftablescience.VPKEdit
+PackageVersion: 4.0.0
+InstallerType: zip
+NestedInstallerType: nullsoft
+NestedInstallerFiles:
+- RelativeFilePath: VPKEdit-4.0.0-win64.exe
+FileExtensions:
+- .bsp
+- .vpk
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 674.)